sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -65,6 +65,20 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
   import hiveContext._
   import spark.implicits._
 
+  test("query global temp view") {
+    val df = Seq(1).toDF("i1")
+    df.createGlobalTempView("tbl1")
+    checkAnswer(spark.sql("select * from global_temp.tbl1"), Row(1))
```

Comment:
Thanks @cloud-fan.  I have changed the test to get the global_temp db from the conf.   spark.conf.get("spark.sql.globalTempDatabase”)

It looks like there are two ways to get it: 
- One is using the conf
- Accessing the spark.sharedState.globalTempViewManager.database

In the test, I am now getting it from the conf.   Please see if this is ok.  Thanks.